packages/hash/api/src/graphql/typeDefs/knowledge/block.typedef.ts

```diff
@@ -25,6 +25,10 @@ export const knowledgeBlockTypedef = gql`
     """
     ownedById: ID!
     """
+    Alias of ownedById - the id of the account that owns this entity.
+    """
+    accountId: ID!
```

Comment:
Should we mark these fields as deprecated? Something like:
```suggestion
    accountId: ID! @deprecated(reason: "accountId is deprecated. Use ownedById instead.")
```